### PR TITLE
(DOCSP-9714): Perform CRUD Operations in a VSCE Playground

### PR DESCRIPTION
**Pull request by jwilliams-mongo**
[JIRA](https://jira.mongodb.org/browse/DOCSP-9714)
Stage
- [Perform CRUD Operations](https://docs-mongodbcom-staging.corp.mongodb.com/6dcbdc2/mongodb-vscode/docsworker/DOCSP-9714/crud-ops)
- [Create Documents](https://docs-mongodbcom-staging.corp.mongodb.com/6dcbdc2/mongodb-vscode/docsworker/DOCSP-9714/create-document-playground)
- [Read Documents](https://docs-mongodbcom-staging.corp.mongodb.com/6dcbdc2/mongodb-vscode/docsworker/DOCSP-9714/read-document-playground)
- [Update Documents](https://docs-mongodbcom-staging.corp.mongodb.com/6dcbdc2/mongodb-vscode/docsworker/DOCSP-9714/update-document-playground)
- [Delete Documents](https://docs-mongodbcom-staging.corp.mongodb.com/6dcbdc2/mongodb-vscode/docsworker/DOCSP-9714/delete-document-playground)

### CODE REVIEW

**Comment by jwilliams-mongo** [comment]
Hi @mmarcon can you review this PR for the VSCE docs? Thank you!

File: source/read-document-playground.txt [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+To read one document, use the following syntax in your Playground:
+
+.. code-block:: javascript
+
+   db.collection.findOne(
+      { <query> },
+      { <projection> }
+   )
+
```
Comment (jeff-allen-mongo) [inline source/read-document-playground.txt:50]: I'd suggest including the behavior that occurs when multiple documents are matched.

```
... returns the first document according to the natural order which reflects the order of documents on the disk.
```

https://docs.mongodb.com/manual/reference/method/db.collection.findOne/#db.collection.findOne

File: source/crud-ops.txt [hunk 2]
```diff
@@ -6,8 +6,41 @@ Perform CRUD Operations
...
-   :depth: 1
-   :class: singlecol
+Documents are individual records in a MongoDB collection and are the 
+basic unit of data in MongoDB.
+
+You can use a MongoDB Playground to perform CRUD (create, read, update,
+and delete) operations on documents in a collection on a 
+:doc:`connected deployment </connect>`.
+
+You use the :manual:`MongoDB CRUD Operators </crud>` and
```
Comment (jeff-allen-mongo) [inline source/crud-ops.txt:16]: This feels like it should either be "You can use" or "Use".

File: source/crud-ops.txt [hunk 3]
```diff
@@ -6,8 +6,41 @@ Perform CRUD Operations
...
+
+You can use a MongoDB Playground to perform CRUD (create, read, update,
+and delete) operations on documents in a collection on a 
+:doc:`connected deployment </connect>`.
+
+You use the :manual:`MongoDB CRUD Operators </crud>` and 
+:manual:`shell methods </reference/method>` to interact with your 
+databases in MongoDB Playgrounds.
+
+.. note::
```
Comment (jeff-allen-mongo) [inline source/crud-ops.txt:20]: Nice note. Really helps highlight the use case for playgrounds.

**Pull request merged**